deployer.rb

```diff
@@ -70,8 +71,14 @@ def client
     @client ||= Octokit::Client.new(:access_token => ACCESS_TOKEN)
   end
 
-  def leankit_ticket_present?(branch_name)
-    !!(branch_name =~ /(\d{6,10})/)
+  def leankit_ticket_present?(pull_request)
+    commit_messages(pull_request).any? do |comment|
+      leankit?(comment)
+    end
+  end
+
+  def leankit?(comment)
```

Comment:
I read this as "is this a leankit comment". Maybe is should read more like "is a leankit card identifier present in the comment".

Reply:
changed to `leankit_identifiable`